cdc/scheduler/internal/info_provider.go

```diff
@@ -32,18 +32,4 @@ type InfoProvider interface {
 	// TODO: Remove the API, as capture no longer has local checkpoint ts and
```

Comment:
@overvenus Do we need to remove it this time?

Reply:
Yes, it's dead code now.

Reply:
Removed! Thanks!